erpnext/education/doctype/student/student.py

```diff
@@ -22,6 +22,10 @@ def validate(self):
 			self.update_student_name_in_linked_doctype()
 
 	def validate_dates(self):
+		for sibling in self.siblings:
+			if sibling.date_of_birth and getdate(sibling.date_of_birth) >= getdate(today()):
```

Comment:
```suggestion
			if sibling.date_of_birth and getdate(sibling.date_of_birth) >= getdate():
```
getdate() by default gives today's date if nothing is passed